tests/integration_tests/without_db/test_download_pretrained_model.py

```diff
@@ -0,0 +1,22 @@
+import json
+import tempfile
+import transformers
+from pathlib import Path
+
+
+def test_download_pretrained_model():
```

Comment:
Add a comment, regarding our intention with this test which is "Checking expected behavior of the 3rd party api"